src/components/Button/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+type childernProviderProps = {
```

Comment:
Naming convention - this should simply be `ButtonProps`. Use Code Shaper to scaffold your classes so that you don't have to worry about these kinds of details.